src/Function/Bundles.agda

```diff
@@ -400,6 +400,15 @@ module _ (From : Setoid a ℓ₁) (To : Setoid b ℓ₂) where
   module SplitSurjection (splitSurjection : SplitSurjection) =
     LeftInverse splitSurjection
 
+------------------------------------------------------------------------
+-- Inline bbreviations for oft-used items
+------------------------------------------------------------------------
+
+-- Since ⟶ is taken below, use a thick arrow instead
+infixr 0 _➙_
+_➙_ : Setoid a ℓ₁ → Setoid b ℓ₂ → Set _
+_➙_ = Func
```

Comment:
So it would be kind of nice if whatever notation we chose for this generalised to the other bundles in this file? It would also be nice if it harmonised with the notation directly below. Would you consider `⟶_s` (subscript)? That would would harmonise with the subscript `s` used to indicate setoid in e.g. https://github.com/agda/agda-stdlib/blob/660d9834670cd8a9afa9d88f3fdf9ab05ee58195/src/Data/Product/Relation/Binary/Pointwise/NonDependent.agda#L183

Reply:
Yes, I would be fine with that.

Reply:
Done.